src/python/bot/minimizer/js_minimizer.py

```diff
@@ -109,13 +149,18 @@ def _execute(self, data):
 
         # 2) Remove comma and right-hand-side.
         #    e.g.: f(crash(), whatever) -> f(crash())
-        if index + 1 < len(testcase.tokens):
-          hypothesis = [index, index + 1]
-          testcase.prepare_test(hypothesis)
 
-      # Keep track of where the previous line started.
-      elif token.endswith('\n'):
-        last_start_of_line_token = index + 1
+        # Find the next non whitespace token after the comma.
+        i = 1
+        whitespace = []
+        while (index + 1 < len(testcase.tokens) and
+               not testcase.tokens[index + i].strip()):
+          whitespace.append(index + i)
+          i += 1
+
+        if index + i < len(testcase.tokens):
```

Comment:
If the loop above is converted to a for loop this could be moved inside. I'm imagining something like:

```
whitespace = []
for whitespace_index in range(index + 1, len(testcase.tokens)):
  if testcase.strip():
    whitespace.append(whitespace_index)
    continue

  # We found a next non-whitespace after the comma. Prepare a hypothesis.
  ...
```

Reply:
See above